### ISSUE TITLE
Superseded web.config files (a cause for false positives)

### ISSUE THREAD

**Issue by mrstarware** [security-code-scan/security-code-scan#170]
**Environment (please complete the following information):**
 - Version [e.g. 3.3.0] 3.5.3
 - Branch [SCS or SCS2017] SCS 2017
 - Visual Studio [e.g. 2019 (16.3.6)] 16.6.0
 - OS: [e.g. Windows, Linux] Windows 10

**Describe the bug**
If a web.config is superseded by a different web.config then the subordinate config is still being scanned independently. 

**Repro**
If I have two web configs in a project and one of them is at 
path1/web.config and the other is at path1/path2/web.config
Then the analyzer does not consider the web.config files together
therefore it is possible to receive a false positive of warnings such as SCS0023
because the primary config has the viewStateEncryptionMode="Always" setting and the other does not.

**Comment by JarLob** [comment]
Yes, web.config rules are more complex than currently supported. See for example https://weblogs.asp.net/jongalloway/10-things-asp-net-developers-should-know-about-web-config-inheritance-and-overrides
Also there are xml transformations in place. Ideal solution probably should use https://github.com/nil4/dotnet-transform-xdt before analysis too.
